packages/mermaid/src/docs/.vitepress/redirect.ts

```diff
@@ -0,0 +1,65 @@
+export const getBaseFile = (link: string): string => {
+  const url = new URL(link);
+  if (
+    (url.hostname !== 'mermaid-js.github.io' && url.hostname !== 'localhost') ||
+    url.pathname !== '/mermaid/'
+  ) {
+    throw new Error('Not mermaidjs url');
+  }
+  const hash = url.hash
+    .toLowerCase()
+    .replace('.md', '')
+    .replace(/^#\/?/g, '')
+    .replace(/^\.\//g, '')
+    .split('?')[0];
+  return hash;
+};
+
+const redirectMap: Record<string, string> = {
+  '8.6.0_docs': '',
+  accessibility: 'config/theming',
+  breakingchanges: '',
+  c4c: 'syntax/c4c',
+  classdiagram: 'syntax/classDiagram',
+  configuration: 'config/configuration',
+  demos: 'misc/integrations',
+  development: 'community/development',
+  directives: 'config/directives',
+  entityrelationshipdiagram: 'syntax/entityRelationshipDiagram',
+  examples: 'syntax/examples',
+  faq: 'misc/faq',
+  flowchart: 'syntax/flowchart',
+  gantt: 'syntax/gantt',
+  gitgraph: 'syntax/gitgraph',
+  integrations: 'misc/integrations',
+  'language-highlight': '',
+  markdown: '',
+  mermaidapi: 'config/usage',
+  mermaidcli: 'config/mermaidCLI',
+  mindmap: 'syntax/mindmap',
+  'more-pages': '',
+  'n00b-advanced': 'config/n00b-advanced',
+  'n00b-gettingstarted': 'intro/n00b-gettingStarted',
+  'n00b-overview': 'community/n00b-overview',
+  'n00b-syntaxreference': '',
+  newdiagram: 'community/newDiagram',
+  pie: 'syntax/pie',
+  plugins: '',
+  quickstart: 'intro/n00b-gettingStarted',
+  requirementdiagram: 'syntax/requirementDiagram',
+  security: 'community/security',
+  sequencediagram: 'syntax/sequenceDiagram',
+  setup: '',
```

Comment:
Should we point this to <http://localhost:5173/mermaid/config/setup/README.html>, aka the following page:

![image](https://user-images.githubusercontent.com/19716675/202269949-61b58e31-4bf2-4745-a8ad-cb0de62a01e3.png)

It's currently very blank, but it where the Mermaid API Configuration sidebar link now points to.

```suggestion
  setup: 'config/setup/README',
```